Implement Basic One-Way switch sensor

nkendoka opened the pull request.

Commit:
Implement Basic One-Way switch sensor

nahueltaibo commented on src/Robot.Drivers/Switches/SwitchSensorEventArgs.cs:
```diff
@@ -0,0 +1,15 @@
+        /// <summary>
+        /// Switch position
+        /// </summary>        
+        public bool On { get; set; }
```
Is there any reason why the SwitchMessage has an On property and the ISwitchSensor one named IsOn?

nkendoka replied:
Semantically is no difference :)
I was just thinking that EventArg is more related to the state transfer and is like a flag 'ON',  SwitchSensor on the other hand has a real exposed property => IsOn.

But you are right, there is no real reason and is confusing. I am going to make it **IsOn** to keep consistent.

nkendoka committed:
Added SwitchSensorDriver

nahueltaibo committed:
Update Robot.MessageBus.csproj

Removing this package reference as its not needed (and messaging library should be unaware of GPIO)
